run_unittests.py

```diff
@@ -2142,7 +2147,9 @@ def build_shared_lib(self, compiler, source, objectfile, outfile, impfile, extra
                 '/NOLOGO', '/DLL', '/DEBUG', '/IMPLIB:' + impfile,
                 '/OUT:' + outfile, objectfile]
         else:
-            extra_args += ['-fPIC']
+            if not (compiler.compiler_type.is_windows_compiler or
```

Comment:
__[Flake8]__ <sub>posted by Sider</sub>

[W504] line break after binary operator

Reply:
> the solution is to disable W504 in setup.cfg?

... which was done in #4389